src/lib/store/AuthModel.ts

```diff
@@ -321,6 +323,10 @@ export const getAuthModel = (): AuthModel => ({
           })
         }
 
+        if (onboardingState === "complete") {
```

Comment:
issue: I think this should be for users who installed the app and already have accounts
```suggestion
        if (onboardingState === "complete" || onboardingState === "none") {
```

Reply:
right. doesnt that mean that the onbaordingState will be "complete"? if we already have an account 🤔.

if having an account means you might have "none" there, then ok.